packages/core/src/quote/Quote.ts

```diff
@@ -74,16 +73,16 @@ export async function createAttesterSignedQuote(
     throw new SDKErrors.QuoteUnverifiableError()
   }
 
-  const signature = await Did.signPayload(
-    quoteInput.attesterDid,
-    Crypto.hashObjectAsStr(quoteInput),
-    sign
-  )
+  const signature = await sign({
+    data: Crypto.coToUInt8(Crypto.hashObjectAsStr(quoteInput)),
```

Comment:
Could we do this here and elsewhere?
```suggestion
    data: Crypto.encodeObjectAsStr(quoteInput),
```

Reply:
so not hashing? maybe?

Reply:
Oh wait, I misread the code… Need to keep hashing,  it’s just that hashObjectAsStr converts to hex only for us to convert it back to UIntArray immediately. How about adding `hashObjectAsHex`? Maybe even removing `hashObjectAsStr`?

Reply:
I was also thinking about having such a function. It would be `hashObjectAsU8a` though

Reply:
how about splitting into `sortAndStringify` and `Crypto.hash` / `Crypto.hashString` instead of having another two functions to cover these combinations?

Reply:
I like it